Canvas.cpp

```diff
@@ -12,17 +14,26 @@ Canvas::Canvas(QWidget *parent) : QWidget(parent)
 Canvas::~Canvas()
 {
     // Free all shapes when deleted. Vector class will delete dynamic array
-    for (auto shapePtr : this->shapes)
-    {
-        delete shapePtr;
-    }
+    this->clearShapes();
 }
 
 Vector<Shape*> Canvas::getShapes()
 {
     return this->shapes;
 }
 
+void Canvas::drawShapesInFile(QString &fileDir)
+{
+    FileHandler fileUtil;
```

Comment:
I don't know how this is valid. ``FileHandler`` is part of the ``jbrush`` namespace but there is no ``jbrush::FileHandler``. I don't see a ``using namespace jbrush;`` either.

Reply:
Yeah its because Canvas is a part of the same namespace